File: samples/WexflowTesting/ExecCs_src/Program2.cs
using System;
using System.IO;

namespace ExecCsTest
{
    class Program2
    {
        static void Main(string[] args)
        {
            try
            {
                File.Copy(@"c:\WexflowTesting\file2.txt", @"c:\WexflowTesting\ExecCs_dest\file2.txt", true);
                Console.WriteLine("File copied!");
            }
            catch (Exception e)
            {
                Console.WriteLine(e);
            }
        }
    }
}

File: samples/WexflowTesting/ExecVb_src/Program2.vb
Imports System
Imports System.IO

Namespace ExecCsTest
    Module Program2
        Sub Main(args As String())
            Try
                File.Copy("c:\WexflowTesting\file2.txt", "c:\WexflowTesting\ExecVb_dest\file2.txt", True)
                Console.WriteLine("File copied!")
            Catch e As Exception
                Console.WriteLine(e)
            End Try
        End Sub
    End Module
End Namespace
